src/Login/LoginForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from 'react'
+import PasswordToggle from "../common/PasswordToggle";
+import {post} from "../utils/http";
+
+type LogInfo = {
+    email: string,
+    password: string
+}
+
+type Props = {
+    whereTo: string
+}
+
+function LoginForm(props: Props){
+
+    const [PasswordInputType, ToggleIcon] = PasswordToggle();
+    const [logInfo, setLogInfo] = useState<LogInfo>({email: "", password: ""});
+    const [error, setError] = useState<string>("")
+
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        if (logInfo.password === "") {
+            setError("Las credenciales ingresadas no son correctas");
+        } else if (logInfo.email === "") {
+            setError("Las credenciales ingresadas no son correctas");
+        } else {
+            const promise = post("auth/", {
+                    email: logInfo.email,
+                    password: logInfo.password
+                },
+                {headers: {"Content-Type": "application/json"}, noAuth: true});
+
+            promise.then(res => {
+                setError("");
+                localStorage.setItem('token', res.accessToken.token);
+                localStorage.setItem('admin', res.admin);
+                window.history.pushState("", "", props.whereTo);
+                window.location.reload();
+            })
+                .catch(() => {
+                    setError("Las credenciales ingresadas no son correctas")
+                })
+        }
+    }
+
+    const changeInfo = (value: string, num: number) => {
+        if(num === 1){
+            setLogInfo({email:value, password: logInfo.password});
+        } else setLogInfo({email:logInfo.email, password: value});
+    }
+
+
+
+    return (
+        <div>
+            <div className="error" hidden={error === ""}><h3 className="error-message">{error}</h3></div>
```

Comment:
You might want to start abstracting components. React is based on reusable components, the main idea is you make a component that would be used in other places. 

In this case you have your error box is used here and also on the signup screen and may be somewhere else. So the best way to do this (the "react" way :sunglasses:) is to create a separate component. Something like this:
```
import React from "react";

interface Props {
    error: string;
    show: boolean;
}

const ErrorBox = (props: Props) => {
    const {error, show} = props;

    return (
        <div className={`error-box ${show ? 'show': 'hide'}`}>
            {error && <span className={"error-text"}>{error}</span>}
        </div>
    )
};

export default ErrorBox;
```

You can add your custom styles to this <ErrorBox/> component and this can be use across the app very easily.

This way you gain all the benefits:
- Easy reusable
- You don't have to copy the style from somewhere else
- Yor code is smaller and more readable